### PR TITLE
Configure New Relic app names

### PR DESCRIPTION
**Pull request by charlie-retzler**

### CODE REVIEW

File: Gateway/crds-angular/Web.Release.config [hunk 1]
```diff
@@ -15,12 +15,13 @@
     </connectionStrings>
   -->
   <appSettings>
-	<add key="CORS" value="https://www.crossroads.net,https://prod.crossroads.net" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="ChangePassword" value="475" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="MyProfile" value="474" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="TokenURL" value="https://admin.crossroads.net/ministryplatform/oauth/token" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="BaseUrl" value="www.crossroads.net" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-	<add key="BaseMPUrl" value="admin.crossroads.net" xdt:Transform="Replace" xdt:Locator="Match(key)" />
+    <add key="NewRelic.AppName" value="Gateway Prod" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
```
Comment (samueljmello) [inline Gateway/crds-angular/Web.Release.config]: i hate your indentation here

File: Gateway/crds-angular/Web.Demo.config [hunk 2]
```diff
@@ -4,12 +4,13 @@
 
 <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
   <appSettings>
-	<add key="CORS" value="https://demo.crossroads.net" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="ChangePassword" value="475" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="MyProfile" value="474" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="TokenURL" value="https://admindemo.crossroads.net/ministryplatform/oauth/token" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
-	<add key="BaseUrl" value="demo.crossroads.net" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-	<add key="BaseMPUrl" value="admindemo.crossroads.net" xdt:Transform="Replace" xdt:Locator="Match(key)" />
+    <add key="NewRelic.AppName" value="Gateway Demo" xdt:Transform="Replace" xdt:Locator="Match(key)"/>
```
Comment (samueljmello) [inline Gateway/crds-angular/Web.Demo.config]: i hate your indentation here

File: Gateway/crds-angular/Web.Release.config (hunk 1 above)
Comment (charlie-retzler) [inline Gateway/crds-angular/Web.Release.config]: Fixed

File: Gateway/crds-angular/Web.Demo.config (hunk 2 above)
Comment (charlie-retzler) [inline Gateway/crds-angular/Web.Demo.config]: Fixed

**Pull request merged**